cita-executor/core/src/cita_executive.rs

```diff
@@ -0,0 +1,30 @@
+// CITA
```

Comment:
No need to add this.
Change the GPL part if you want to.

Reply:
All these license docs could be removed after our task.

Reply:
Then, why do we add it first?

Reply:
Adding now or Removing together later is accecpted to me.

Reply:
okay, delete it.